src/plugin.js

```diff
@@ -90,7 +90,8 @@ function dispatchEvent(chart, listeners, label) {
   }
 }
 
-function dispatchMoveEvents(chart, listeners, previous, label) {
+function dispatchMoveEvents(chart, listeners, labels, event) {
+  const {previous, label} = labels;
```

Comment:
This line should be removed according my comment above but keep in mind that this plugin doesn't rely on transpilation. So to be consistent with the rest of the code, use `var` and avoir destructuring objects. I'm actually surprised that you didn't get any ESLint error.

Reply:
The same reason of the first item

Reply:
done